### PR TITLE
feat(services) change color for 'Partially degraded'

### PR DESCRIPTION
**Pull request by lobkovilya**
Before:
image

After:
image

### CODE REVIEW

**Review by nickolaev — APPROVED** [review]

File: src/assets/styles/components.scss [hunk 1]
```diff
@@ -20,11 +20,11 @@
 }
 
 .entity-status.is-degraded {
-  color: var(--yellow-300);
+  color: #ff8040;
```
Comment (tharun208) [inline src/assets/styles/components.scss:23]: just a thought, `--yellow-400` won't work for this?

Comment (lobkovilya) [reply src/assets/styles/components.scss:23]: not really :D 
image

Comment (tharun208) [reply src/assets/styles/components.scss:23]: I referred kongponents for the color code - https://kongponents.konghq.com/style-guide/colors.html#colors for `yellow-400`.

Comment (lobkovilya) [reply src/assets/styles/components.scss:23]: Oh I see, that's weird, I think `kuma-gui` depends on the previous version of kongponents, there are only 4 yellow colors like on the screenshot

**Pull request merged**